test/shell/test_macros.sh

```diff
@@ -5,7 +5,7 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 runner=$(get_test_runner "${1:-local}")
 
 incorrect_macro_user_does_not_build() {
-  (! bazel build //test/macros:incorrect-macro-user) |&
+  (! bazel build //test/macros:incorrect-macro-user) 2>&1 |
```

Comment:
This was invalid on my mac `GNU bash, version 3.2.57(1)-release (arm64-apple-darwin24)`

Reply:
Heh, yeah, I forgot to mention that. I just fixed that locally, too. Wouldn't hurt to put a `!#/usr/bin/env bash` at the top, either.